Wrap into a gem

mx4492 opened the pull request:
Now you can do `gem install git-game`.

mnvr committed:
Wrap into a gem

mx4492 commented:
@jsomers If you provide me with the email address for your rubygems.org account, I'll add you as a owner.

ayushgoel commented:
Nice :+1: 
Thanks!

pklopping commented:
(Y)

jsomers commented:
@mx4492 This is sick! Rubygems account email is my github handle at gmail.

Would you mind fixing merge conflicts on this as well? (I'm sure they're trivial.)

Also, is there any relationship between the version given in the gemspec and the GitHub release version number? If so, I've now bumped it to 1.2. Thanks!

mnvr committed:
Merge remote-tracking branch 'upstream/master'

mnvr committed:
Document steps required for releasing a new version

mnvr committed:
Development.md: Fix grammar

mx4492 commented:
Fixed the conflicts.

I've added you as an owner. Since you've already pushed the 1.2 tag to Github, it would be incorrect to push a new gem with the same version number but different source, so I suggest that you skip 1.2 for Rubygems and instead release a new version 1.3 (on both Github and Rubygems) once this is merged.

> Also, is there any relationship between the version given in the gemspec and the GitHub release version number?

No. There isn't. However, if you have bundler installed on your system, then there is a handy rake task to ensure that they remain in sync. I've listed the steps to release a new version in the `Development.md` file in this PR (I can revert that commit if you think documenting these steps is unnecessary).
